go.mod

```diff
@@ -364,3 +364,5 @@ replace github.com/unrolled/secure => github.com/DeepDiver1975/secure v0.0.0-202
 // exclude the v2 line of go-sqlite3 which was released accidentally and prevents pulling in newer versions of go-sqlite3
 // see https://github.com/mattn/go-sqlite3/issues/965 for more details
 exclude github.com/mattn/go-sqlite3 v2.0.3+incompatible
+
+replace github.com/cs3org/reva/v2 => github.com/butonic/reva/v2 v2.19.1-0.20240521095806-57743c6767f7
```

Comment:
probably a noob question, but why to `butonic`? And could you please use the most recent edge commit to get even more changes in? :-)

Reply:
hehe, because we are testing one of my ocis PRs in this PR ;-)